mtp_api/apps/core/management/commands/dump_for_ap.py

```diff
@@ -96,8 +130,13 @@ def serialise(self, record: Credit):
             'Prisoner number': record.prisoner_number or 'Unknown',
             'Prisoner name': record.prisoner_name or 'Unknown',
             'Prison': record.prison.short_name if record.prison else 'Unknown',
+            'Owner username': record.owner.username if record.owner else 'Unknown',
+            'Blocked': record.blocked,
             'Status': status,
             'NOMIS transaction': record.nomis_transaction_id,
+            'Security check status': security_check_status,
+            'Security check codes': security_check_rules,
+            'Exported at': self.exported_at_local_time,
```

Comment:
no big deal, but perhaps putting this field first makes sense since it's common to all records and _types_ of records too. your call :)